terraform/environments/development-eu-west-2/devops1/vars

```diff
@@ -0,0 +1,20 @@
+aws_bucket = "development-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-development-terraform-state-london"
+aws_region = "eu-west-2"
+environment = "devops1"
+deploy_to = "development"
+state_prefix = "env:/development"
+workspace_key_prefix = "test-data-generator-stack"
+state_file_name = "test-data-generator.tfstate"
+
+aws_profile = "development-eu-west-2"
+ec2_instance_type = "t3.medium"
+
+# Certificate for https access through ALB
+ssl_certificate_id = "arn:aws:acm:eu-west-2:169942020521:certificate/248f5001-3d30-4d90-bec3-023e5ab1f572"
+zone_id = "Z2KSI4Z5ZN9NT0"
+#top_level_domain = ".devops1.aws.chdev.org"
```

Comment:
If this isn't needed it should come out

Reply:
Yeah, good shout. Unused "top_level_domain" var now removed (from all environments).